src/Controller/Api/DatasetSubmissionController.php

```diff
@@ -27,23 +29,20 @@ class DatasetSubmissionController extends EntityController
      *
      * @param Request $request The request object.
      *
-     * @ApiDoc(
-     *   section = "Dataset Submission",
-     *   input = {
-     *     "class": "Pelagos\Bundle\AppBundle\Form\EntityCollectionType",
-     *     "name": "",
-     *     "options": {
-     *       "label": "Dataset Submission",
-     *       "data_class": "Pelagos\Entity\DatasetSubmission"
-     *     }
-     *   },
-     *   output = "array<Pelagos\Entity\DatasetSubmission>",
-     *   statusCodes = {
-     *     200 = "The requested collection of Dataset Submissions was successfully retrieved.",
-     *     500 = "An internal error has occurred.",
-     *   }
+     * @Operation(
+     *     tags={"Dataset Submission"},
+     *     summary="Get a collection of Dataset Submissions.",
```

Comment:
I don't know if this ever worked? Dataset submission collection throws an error 
"Could not convert database value \"0001-12-31 18:09:24-05:50:36 BC\" to Doctrine Type datetimetz. Expected format: Y-m-d H:i:sO"
And we aren't using this anywhere in our code base

Reply:
This is because of a zero date that is being converted to UTC (less time) and makes it a weird date with BC in it. (Before Christ).